public/app/plugins/panel/gauge/DisplayValueEditor.tsx

```diff
@@ -0,0 +1,64 @@
+// Libraries
+import React, { PureComponent } from 'react';
+
+// Components
+import { FormField, FormLabel, PanelOptionsGroup, UnitPicker } from '@grafana/ui';
+
+// Types
+import { DisplayValueOptions } from '@grafana/ui';
+
+const labelWidth = 6;
+
+export interface Props {
+  options: DisplayValueOptions;
+  onChange: (options: DisplayValueOptions) => void;
+}
+
+export class DisplayValueEditor extends PureComponent<Props> {
```

Comment:
SingleStatValueEditor? can this also include stat?

Reply:
no :) for reasons above.

without 'stat' it can be reused for the table cell properties as well.  and anywhere that just needs to take a value<any> and show it in the UI safely as a string/number

We could write it so that you could pass in the stat selection component though...

Reply:
actually... yes, the editors could/should be as custom as we want them to be for each plugin.

My goal with this was to write a single edit box that could be used in: gauge & table, but there is no reason it needs to be reused (aside from lots of boilerplate code)

However, I still think it makes sense to separate options for what the data is (stat) from how to display it: prefix/suffix/units/etc